lib/appium_console.rb

```diff
@@ -50,6 +50,11 @@ def start
 
           $stdout.puts "pry #{cmd.join(' ')}"
         end
+
+        Pry.hooks.add_hook(:after_session, "Release session hook") do |output, binding, pry|
+          output.puts "Releasing appium, please wait..."
```

Comment:
I think this is a bit more in line with the [JSONWireProtocol language](https://w3c.github.io/webdriver/webdriver-spec.html)

- `Closing appium session...`